### DISCUSSION TITLE
[4.4] Entrify a globalset - losing permissions

### DISCUSSION THREAD

**Discussion by robin-gauthier** [craftcms/cms#12849] [Ideas] [answered]
When you run the entrify/globalset command, the new single is created. But if you are not in the admin users group, the permission are not you had on the globalset is not transferred to the new section.

It would be nice if the command line could automatically copy the permissions so the admin doesn't have to do it manually in each users group.

**Comment by brandonkelly** [comment, marked as answer]
Good call—added for the next release! (73771c5811656a3139087a370e3e729f1d7ff96f)

`entrify/categories` will now register the following permissions, based on the preexisting user/user group permissions in place:

Category Permission | Entry Permissions
-- | --
View categories | View entries, View other users’ entries
Save categories | Create entries, Save entries, Save other users’ entries
Delete categories | Delete entries, Delete other users’ entries
View other users’ drafts | View other users’ drafts
Save other users’ drafts | Save other users’ drafts
Delete other users’ drafts | Delete other users’ drafts

And for `entrify/global-set`:

Global Set Permission | Entry Permissions
-- | --
Edit “[title]” | View entry, Save entry, View other users’ drafts, Save other users’ drafts, Delete other users’ drafts

(Tag groups don’t have permissions, so we don’t have anything to go off of for them; they will need to be added manually.)

**Reply by brandonkelly** [reply]
4.4.2 is out with that change.
